beacon-chain/rpc/apimiddleware/custom_handlers.go

```diff
@@ -213,7 +213,8 @@ func handlePostSSZ(m *apimiddleware.ApiProxyMiddleware, endpoint apimiddleware.E
 	return true
 }
 
-func sszRequested(req *http.Request) (bool, error) {
+// SszRequested returns a bool value if the ssz type is requested.
+func SszRequested(req *http.Request) (bool, error) {
```

Comment:
Let's maybe move this function to `/network` (how about a `reader.go` file?) because it doesn't make sense to import the middleware package in the HTTP handler. We shouldn't need the middleware at all when writing HTTP handlers.